plugins/plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/editor/JavaCodeAssistClient.java

```diff
@@ -116,8 +116,7 @@ public String getProposalDocUrl(int id, String sessionId) {
         return newPromise(new AsyncPromiseHelper.RequestCall<List<Change>>() {
             @Override
             public void makeCall(AsyncCallback<List<Change>> callback) {
-                String url = appContext.getDevMachine().getWsAgentBaseUrl() + "/code-formatting/format?offset=" + offset +
-                             "&length=" + length;
+                String url = appContext.getDevMachine().getWsAgentBaseUrl() + "/java/code-formatting/format?offset=" + offset + "&length=" + length;
```

Comment:
should we use a constant prefix for all the paths of this class ?
constant being "/java/code-formatting/" to avoid to miss some endpoint in a future